common/cache/metricsScopeCache_test.go

```diff
@@ -47,6 +49,7 @@ func TestDomainMetricsCacheSuite(t *testing.T) {
 
 func (s *domainMetricsCacheSuite) SetupTest() {
 	s.Assertions = require.New(s.T())
+	s.metricsClient = metrics.NewClient(tally.NoopScope, metrics.Frontend)
```

Comment:
there is a tally-test-scope which is probably what should be used for most of these tests.

but almost certainly beyond the scope of this PR 👍